CHANGES.rst

```diff
@@ -6,6 +6,14 @@ Unreleased
 ==========
 
 
+2024-01-12 1.24.7
+=================
+
+- Fixed grammar in ES version of ``DISCOURSE_MSG``. Thanks, @hlcianfagna.
```

Comment:
is `DISCOURSE_MSG` a user facing variable? if not please rephrase, using something that a plain user understands.

Reply:
I amended this now, feel free to unresolve if you think it needs further changing